src/main/java/io/stargate/sgv2/jsonapi/config/OperationsConfig.java

```diff
@@ -85,15 +85,18 @@ public interface OperationsConfig {
   @WithDefault("20")
   int maxDocumentInsertCount();
 
-  /**
-   * @return Maximum size of _id values array that can be sent in $in operator <code>100</code>
-   *     command.
-   */
+  /** @return Maximum size of values array that can be sent in $in operator */
   @Max(100)
   @Positive
   @WithDefault("100")
   int maxInOperatorValueSize();
 
+  /** @return Maximum size of values array that can be sent in $in operator */
```

Comment:
that can be sent in `$in` and `$nin` operator

Reply:
have maxNinOperatorValueSize() just below,
Or do you mean we should have one instead of two